pages/api/contactForm.js

```diff
@@ -0,0 +1,30 @@
+import axios from "axios";
+
+export default async function handler(req, res) {
+  if (req.method === "POST") {
+    const { name, email, message } = req.body;
+
+    const webhookURL =
+      "https://discord.com/api/webhooks/1097847480697225226/8smtlEwEhZDyNFocN089-y2YFJqF34dAM7TLdP6X7nM3XeIC6qBLhMjO5bkCc34yWgeM";
```

Comment:
The discord docs suggest that the last part of this url is your discord token: https://discord.com/developers/docs/resources/webhook#get-webhook-with-token

The token part needs to go into your `.env.local` file and into the vercel deployment settings.

> **Warning**
> This means that it is essentially a password for your discord server. Which you don't want to have in your code.
> **Please immediately delete this token in your discord (app or server) settings (where ever you created it) and create a new one that wasn't publicly leaked.**

Your code should look like this:
```suggestion
    const webhookURL =
      `https://discord.com/api/webhooks/1097847480697225226/${process.env.DISCORD_WEBHOOK_TOKEN}`;
```

and then in your `.env.local`
```env
DISCORD_WEBHOOK_TOKEN = "8smtlEwEhZDyNFocN089-y2YFJqF34dAM7TLdP6X7nM3XeIC6qBLhMjO5bkCc34yWgeM"
```